python/mlc_llm/serve/engine_base.py

```diff
@@ -462,6 +513,90 @@ def infer_args_under_mode(
     return kv_cache_config, model_max_single_sequence_length
 
 
+def _infer_kv_cache_config_for_rnn_state(  # pylint: disable=too-many-arguments,too-many-locals,too-many-branches,too-many-statements
```

Comment:
Please log out the KV cache config values as in `_infer_kv_cache_config_for_kv_cache`. Here is a pointer PR https://github.com/mlc-ai/mlc-llm/pull/2128